PIM/Overview/01_General.md

```diff
@@ -0,0 +1,22 @@
+# General information
+
+The *Actindo Product Information Management (PIM)* is the one source of truth for collecting, managing and enriching product data and thus the heart of all your sales channels. Developed according to the API-first principle, the *PIM* module can be quickly and easily integrated into existing system landscapes and serves as the basis for efficient unified commerce across all channels.
+
+This documentation refers to *PIM* module version 4.2.0. There may be differences in the UI, features and operation if your *PIM* module is installed in a different version.
+
+
+## Benefits
+
+The *PIM* module offers the following benefits:
+- Flexible integration into existing system landscapes, including data transformation from legacy IT-systems
+- Real-time management and control of product data for all sales channels and marketplaces
+- Central product data hub that breaks down data silos and ensures high data quality
+- Data output individually tailored to each output channel
+
+
+## Prerequisites
```

Comment:
These Prerequisites are only correct for PIM Version 4.2.x, differing Versions of PIM might have other needed Modules